src/router/index.js

```diff
@@ -73,6 +73,30 @@ const routes = [
             }
             // 失物招领组界限 ---------- [完]
         ]
+    },
+    {
+        // 新闻中心
+        path: '/news-center',
+        component: () => import('@/views/Wrapper.vue'),
+        children: [
+            {
+                path: '/',
+                name: 'NewsCenter',
+                component: () => import('@/views/news/news-center/index.vue')
+            }
+        ]
+    },
+    {
+        // 新闻页面
+        path: '/news-detail',
+        component: () => import('@/views/Wrapper.vue'),
+        children: [
+            {
+                path: '/',
+                name: 'Detail',
```

Comment:
```suggestion
                name: 'NewsDetail',
```
使用 `detail` 这个词太宽泛了，要具体点